### PR TITLE
Bump appservice to fix the gitignore issue and remove globby

### PR DESCRIPTION
**Pull request by nturinski**

### CODE REVIEW

File: package-lock.json
Comment (bwateratmsft) [inline package-lock.json:1]: Globby isn't actually being removed in package-lock.json, is that expected?

Comment (bwateratmsft) [reply package-lock.json:1]: It looks like it's still a dependency of `@microsoft/vscode-azext-azureappservice@2.2.7`

Comment (nturinski) [reply package-lock.json:1]: Yeah, we had to leave it as a dev dependency due to this issue: https://github.com/microsoft/vscode-azuretools/pull/1616/files/bd5390300a1e4eeea74b349685b896035fe1ad09#r1376208534

**Review by bwateratmsft — APPROVED** [review]

**Comment by nturinski** [comment]
Closing in favor of https://github.com/microsoft/vscode-azureappservice/pull/2595

**Pull request closed without merging**